### PR TITLE
Travis Configuration

### PR DESCRIPTION
**Pull request by Awinja-j**
#### This PR introduces Travis Configuration in the `.travis.yml`  file to the repo. 
The configurations include: 
- The version of python being used, 
- Command used to install requirements using  requirements.txt,
- Command used to run the tests that will be written.

### CODE REVIEW

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
So close! So very close... 😞

This - `healthtools/manage.py → manage.py`.. No.

**Comment by DavidLemayian** [comment]
Also, let's add travis build badge to the README.md while we're at it. Bonus points: use the high quality badge creator that's somewhere online (too lazy to go find it right now).

**Comment by Awinja-j** [comment]
@DavidLemayian the manage.py has been moved back to its original folder .

**Comment by DavidLemayian** [comment]
It looks like you've deleted `healthtools/manage.py` all together.

Also, would have been good to see the badge added to README.md even if in "holding" status..

**Comment by DavidLemayian** [comment]
@Awinja-Andela what about `healthtools/manage.py`? You've deleted it..

File: healthtools/manage.py [hunk 1]
```diff
@@ -11,4 +11,4 @@
 
 
 if __name__ == "__main__":
-    app.run()
+     app.run()
```
Comment (DavidLemayian) [inline healthtools/manage.py]: Is this indentation correct?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#travis badge
```
Comment (DavidLemayian) [inline README.md]: Why is this here?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Awinja-j) [reply README.md]: @DavidLemayian i have made changes on this.
